lib/generators/hyrax/work/work_generator.rb

```diff
@@ -34,8 +34,8 @@ def create_indexer
     template('indexer.rb.erb', File.join('app/indexers', class_path, "#{file_name}_indexer.rb"))
   end
 
-  def create_form
-    template('form.rb.erb', File.join('app/forms/hyrax', class_path, "#{file_name}_form.rb"))
```

Comment:
The spec/forms/hyrax/generic_work_form_spec.rb still references `GenericWorkForm` which is no longer being created.

Reply:
Removed the `GenericWorkForm` spec and added in a basic spec for `GenericWorkChangeSet`

Reply:
#1851 added to the backlog to enhance the features for change sets and spec for `GenericWorkChangeSet`